src/server.cc

```diff
@@ -59,6 +59,24 @@ void Server::start_accept() {
           boost::asio::placeholders::error));
 }
 
+// Run the server in threads
+void Server::run() {
+  // Modeled after team AAAAA from class, which was shown in class code reviews
+  
+  // List of unique pointers to each thread
+  std::vector<boost::shared_ptr<boost::thread>> list_of_threads;
```

Comment:
std::unique pointers aren't compatible with the threading from boost, for some reason..